iceoryx_posh/include/iceoryx_posh/runtime/service_discovery.hpp

```diff
@@ -17,6 +17,7 @@
 #ifndef IOX_POSH_RUNTIME_SERVICE_DISCOVERY_HPP
 #define IOX_POSH_RUNTIME_SERVICE_DISCOVERY_HPP
 
+#include "iceoryx_hoofs/cxx/helplets.hpp"
```

Comment:
If you move the `Wildcard_t` into the service description header you could remove this include here.